data/core/macros/interface-utils.cfg

```diff
@@ -201,50 +201,78 @@
     [/color_adjust]
 #enddef
 
-# Flashes the screen with a given color performing ACTION_WML
-# These macros come in WHITE, RED, GREEN and BLUE and can
-# easily be expanded for ORANGE, PURPLE etc.
+#define SCREEN_FADE RED GREEN BLUE DURATION
+    [screen_fade]
+        red={RED}
+        green={GREEN}
+        blue={BLUE}
+        alpha=255
+        duration={DURATION}
+    [/screen_fade]
+#enddef
+
+#define SCREEN_UNFADE DURATION
+    [screen_fade]
+        alpha=0
+        duration={DURATION}
+    [/screen_fade]
+#enddef
+
+#define SCREEN_FADE_OUT
+    {SCREEN_FADE 0 0 0 500}
+#enddef
+
+#define SCREEN_FADE_IN
+    {SCREEN_UNFADE 500}
+#enddef
+
+# Flashes the screen with a given color performing ACTION_WML.
```

Comment:
This documentation comment is not parsed out by wmlscope for whatever reason. Either it needs to be inside the `#define` or the blank line following it needs to be removed.

This is not new in this PR – it was already the case. So you don't need to change it, unless you want to.

Reply:
I also found the scope of this comment unclear. It's referring to several macros in bulk so it's not really directly attached to any of them. Maybe it should just be removed? The individual defines already have comments inside them.